### PR TITLE
MOD landing page

### PR DESCRIPTION
**Pull request by sibyl229**
An independently deployable app for the MOD landing pages. It's a Next.js app that is exported as HTML pages during deployment. 

AWS Amplify is used to deploy and host these pages. An re-deployment is to be triggered when commits are made to the branch. A redeployment can also be triggered with a webhook. These still need to be configured once the PR goes in, because Amplify based deployments are configured per branch.

For testing, run 

```
npm start mod-landing-pages
```

The server will be accessible at:
http://localhost:4200/members/wormbase

### CODE REVIEW

**Comment by adamgibs** [comment]
@sibyl229  This looks cool. I've heard of Next.js but I haven't actually used it. Also, the different mod themes look good. One thing that I'm not sure if it's a small bug or just a temporary placeholder is the `//footer` at the bottom of each page above the footer.

File: apps/mod-landing-pages/pages/[mod].js [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+            eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
+            ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+            aliquip ex ea commodo consequat. Duis aute irure dolor in
+            reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
+            pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
+            culpa qui officia deserunt mollit anim id est laborum.
+          </p>
+        </section>
+      </main>
+      <footer>// Footer</footer>
```
Comment (adamgibs) [inline apps/mod-landing-pages/pages/[mod].js:105]: Temporary placeholder?

Comment (sibyl229) [reply apps/mod-landing-pages/pages/[mod].js:105]: Thanks @adamgibs! Good catch! I will fix that. I was testing the footer in storybook and forgot to move it to the actual page.

**Review by adamgibs — APPROVED** [review]
Looks good!

**Comment by sibyl229** [comment]
Thanks @adamgibs for the speedy check!  I will merge it in

**Pull request merged**